CMSgov/onemac
# Made rai response addl info section use the AdditionalInfoSection component

Story: https://qmacbis.atlassian.net/browse/OY2-22495
Endpoint: See github-actions bot comment

### Details

Format RAI Response addl info same as main content to always show section but display custom message if no content

### Changes

- updated AdditionalInfoComponent to pass a string with content and optional custom id
- updated existing usage of component to pass in only content string
- updated detail section component to use additionalInfo component in the RAI section


### Test Plan

1. Login as state submitter
2. Locate or submit a package with an rai response without additional information
3. verify rai response section of detail view contains additional information section with default text
